src/styles/typography/index.md.njk

```diff
@@ -99,6 +99,27 @@ Use the `govuk-link--no-visited-state` modifier class where it is not helpful to
 
 {{ example({group: "styles", item: "typography", example: "link-no-visited-state", html: true, open: true}) }}
 
+If it's an external link to a non-government website, make that clear in the link text. For example, 'read advice on writing link text from [name of organisation]'. There's no need to say explicitly that you're linking to an external site. [Do not use an external link icon](https://designnotes.blog.gov.uk/2016/11/28/removing-the-external-link-icon-from-gov-uk/).
+
+### Opening links in a new tab
+
+Avoid opening links in a new tab or window. It can be disorienting - and [can cause accessibility problems for people who aren't able to perceive the link visually](https://www.w3.org/TR/WCAG20-TECHS/G200.html).
+
+If you need a link to open in a new tab - for example, to stop the user losing information they’ve entered into a form - then include the words ‘opens in new tab’ as part of the link.
```

Comment:
Is there a reason to prefer ‘opens in a new tab‘ over ‘opens in a new window‘?

Reply:
Most browsers default to a new tab, on mobile I'm not sure a new window is even possible

Reply:
Hi, there were a couple of things I had in mind -

I *think* 'opens in new tab' is the default behaviour in most modern browsers.

Plus it seems better to use one or the other - 'opens in new tab or window' seems excessive. And 'tab' is a little shorter than 'window'.

Does that make sense?

Reply:
Yep, makes sense!